.github/workflows/ci_cd.yml

```diff
@@ -142,6 +142,10 @@ jobs:
       - name: Start API, ingest and index test data
         run: just init
 
+      - name: Smoke test ReDoc site
```

Comment:
🔥

Reply:
When would this fail? On 500 error from the server? Would it fail on main?

Reply:
It fails for any non-success status code from the endpoint. So yes, it fails on main as `curl` returns exit code 22 if the response status code is 500.

Reply:
Hmm, I'm trying this command against `api-dev.openverse.engineering` (which is currently failing to load) and it's passing just fine 😟 When I add the `-L` flag for `curl` it fails as you describe. Do we need that flag here too?